src/App.js

```diff
@@ -1,24 +1,35 @@
-import logo from './logo.svg';
-import './App.css';
+import { Navbar } from "./Components/Navbar";
+import { FeaturedCard } from "./Components/FeaturedCard";
+import { LandingIcon } from "./Components/LandingIcon";
+import { Footer } from "./Components/Footer";
+import { OurServices } from "./Components/OurServices";
+import { Articles } from "./Components/Articles";
+import { LastestsArticles } from "./Components/LastestsArticles";
+import { CarouselSection } from "./Components/CarouselSection";
+import { DownSectionIcons } from "./Components/DownSectionIcons";
 
 function App() {
+  
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+      <div className="mx-auto mt-12 w-4/5">
+        <Navbar />
+        <LandingIcon type="dotts" className="absolute lg:left-0 right-0" />
+        <FeaturedCard />
+        <OurServices />
+      </div>
+      <div className="relative w-screen">
+        <LandingIcon type="dotts" className="absolute right-4" />
+      </div>
+      <div className="mx-[10%] mt-[24%] relative">
```

Comment:
watch out with the `mt-[24%]`, is very unpredictable to use percentage on height, when it is set to auto, like here.
If you have a reeeeeeeallly long text, the height of app would grow a lot, and so will `mt-[24%]`, unless it is grabbing the screen height.
Can't we use a built in class? it looks like we just want a little bit of padding here. We could use mt-[123rm], using rm instead!